ksqldb-common/src/main/java/io/confluent/ksql/function/GenericsUtil.java

```diff
@@ -172,11 +173,15 @@ public static SqlType applyResolved(
     return ImmutableMap.copyOf(mapping);
   }
 
+  // CHECKSTYLE_RULES.OFF: CyclomaticComplexity
   private static boolean resolveGenerics(
       final List<Entry<GenericType, SqlType>> mapping,
       final ParamType schema,
-      final SqlType instance
+      final SqlArgument instance
   ) {
+    final SqlType sqlType = instance.getSqlType();
+
+    // CHECKSTYLE_RULES.ON: CyclomaticComplexity
```

Comment:
Is there a usual pattern for where we turn the checkstyle rules back on?

Reply:
Just following the other patterns, we usually turn it back on right after the method signature, I moved it up one line now